src/arrayMethodSort.js

```diff
@@ -4,8 +4,30 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a.toString() > b.toString()) {
+        return 1;
+      } else if (a.toString() < b.toString()) {
+        return -1;
+      } else {
+        return 0;
```

Comment:
The default compare function is converting values to strings for comparison, which works for string sorting. However, for numerical sorts, it might lead to incorrect results since it compares numbers as strings (e.g., '10' will come before '2'). Consider implementing a more generic default comparison that handles numbers correctly as well.